Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml

```diff
@@ -4,35 +4,35 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/content_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:contentDescription="@string/contact_diary_overview_title">
 
-        <include
-            android:id="@+id/contact_diary_overview_header"
-            layout="@layout/include_contact_diary_overview_header"
+        <androidx.appcompat.widget.Toolbar
```

Comment:
Toolbar in Overview Screen should be completely black instead of grey when using dark mode

Reply:
We have a design for that? The app is now consistent in dark mode, before it was not

Reply:
Ok, I will change it to black for this screen.